Makefile

```diff
@@ -10,10 +10,7 @@ MANINSTALLDIR=${PREFIX}/share/man
 GO_MD2MAN ?= /usr/bin/go-md2man
 
 ifeq ($(DEBUG), 1)
-  GOGCFLAGS += "-N -l"
-else
-  GOGCFLAGS ?= ""
-  DEBUG ?= 0
```

Comment:
and 	
`go build -ldflags "-X main.gitCommit=${GIT_COMMIT}" $(if $(GOGCFLAGS),-gcflags "$(GOGCFLAGS)") -o skopeo ./cmd/skopeo` to also remove the empty -gcflags case.. if you want